yt_dlp/extractor/bilibili.py

```diff
@@ -660,16 +683,248 @@ def get_metadata(page_data):
             return {
                 'page_count': math.ceil(entry_count / page_size),
                 'page_size': page_size,
-                'title': traverse_obj(page_data, ('meta', 'name'))
+                'uploader': self._get_uploader(mid, playlist_id),
+                **traverse_obj(page_data, {
+                    'title': ('meta', 'name', {str_or_none}),
+                    'description': ('meta', 'description', {str_or_none}),
+                    'uploader_id': ('meta', 'mid', {str_or_none}),
+                    'timestamp': ('meta', 'ptime', {int_or_none}),
+                    'thumbnail': ('meta', 'cover', {url_or_none}),
+                })
+            }
+
+        def get_entries(page_data):
+            return self._get_entries(page_data, ('archives', ..., 'bvid', {str}))
```

Comment:
To go along with the suggestion for `BilibiliSpaceListBaseIE._get_entries()`:
```suggestion
        def get_entries(page_data):
            return self._get_entries(page_data, 'archives')
```